docs/documentation.md

```diff
@@ -107,8 +107,8 @@ import {
   //components
   ReactSVGPanZoom,             //Controlled component
   UncontrolledReactSVGPanZoom, //Uncontrolled component
-  
-  //functions to manipulate the state. The signature is quite similar to component methods, expect for the first parameter that is the current value. These are usefull only for some advanced integrations (e.g. reducers). 
+
+  //functions to manipulate the state. The signature is quite similar to component methods, expect for the first parameter that is the current value. These are usefull only for some advanced integrations (e.g. reducers).
```

Comment:
This PR changes a lot of blank line. Can you clean this file and submit only the changed text?